example/README.md

```diff
@@ -1,40 +1,49 @@
-Build Example
-=====================
+# Capacitor Example App
 
-## Build Capacitor Core Module
+This Example app can be used to develop Capacitor. It uses Capacitor to create native iOS and Android apps, the web app itself is an Ionic app, but the important bit is that the native apps are set up in a way that enables development of the native code by loading it from the parent folder instead of as an external dependency.
 
-```
-cd to core
+Note: This project does _not_ use the Capacitor CLI. Please follow the steps below to build, prepare and run the apps.
+
+## 1. Build Capacitor Core Module
+
+Start by building the Capacitor Core Module in `/core`:
 
+```bash
+cd ../core
+
+npm install
 npm run build
 npm link
 ```
 
+## 2. Build Example App
 
-## Build Example App
-
-This will npm link to the Capacitor package, build the app, and copy the build files to the correct `public` directories for both the iOS and Android example apps.
+Switch back over to this example project in `/example` where you first install dependencies and link in the `@capacitor/core` you just built in the step before, then build the app and copy the build files to the correct `public` directories for both the iOS and Android example apps:
 
-```
-cd to example
+```bash
+cd ../example
 
+npm install
 npm link @capacitor/core
+
 npm run build
 npm run copy
 ```
 
+## 3. Build the native Capacitor Apps
 
-## Build iOS App
+Now that everything is in place you can build the native Capacitor Apps:
 
-```
+### a) Build iOS App
+
+```bash
```

Comment:
This is not bash, nor commands, so not sure if should be formated like this.
Also, should open IonicRunner.xcworkspace, not the .xcodeproj

Reply:
- correct @ bash
- `xcodeproj` is what was already there. will fix.